extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Wikia\ContentReview;
+
+class Hooks {
+
+	public static function onGetRailModuleList( Array &$railModuleList ) {
+		global $wgTitle, $wgUser;
+
+		if ( $wgTitle->getNamespace() === NS_MEDIAWIKI && $wgTitle->userCan( 'edit', $wgUser ) ) {
```

Comment:
It's not that important for our code, but at least as a MW standard, it's [preferable](https://github.com/Wikia/app/blob/fae57ff57bdc4b4f8e/includes/Title.php#L947-L949) to use `$wgTitle->inNamespace( NS_MEDIAWIKI );`.